packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -425,7 +425,7 @@ OpenPositions.propTypes = {
     getPositionById: PropTypes.func,
     is_loading: PropTypes.bool,
     is_multiplier: PropTypes.bool,
-    NotificationMessages: PropTypes.node,
+    NotificationMessages: PropTypes.any,
```

Comment:
There shouldn't be more than two prop types passed, do we really need to go with `any`? 🤔

Reply:
@zaki-binary I just updated it.

Reply:
Is it possible to use this prop-types instead? 
```
NotificationMessages: PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.node),
        PropTypes.node
    ])
```

Reply:
@reza-fs seems my last commit not pushed my bad. for this case, as it's a `react component` the only prop that works fine was `func`